src/Files.App/Utils/Storage/Operations/FilesystemHelpers.cs

```diff
@@ -871,7 +871,7 @@ public static bool ContainsRestrictedFileName(string input)
 		/// </summary>
 		public static string GetShortcutNamingPreference(string itemName)
 		{
-			var keyName = @"HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\NamingTemplates";
+			const string keyName = @"HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\NamingTemplates";
```

Comment:
As above

Reply:
And this I agree, should be a `const` field.